cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1410,6 +1410,28 @@ def test_json_responses(self):
         json_response = json.loads(resp.content)
         self.validate_course_xblock_info(json_response, course_outline=True)
 
+    def test_xblock_outline_handler_mongo_calls(self):
+        expected_calls = 5
+        with self.store.default_store(ModuleStoreEnum.Type.split):
```

Comment:
@cahrens Yes and I already did this in https://github.com/edx/edx-platform/pull/8496.